tests/test_plan_stubs.py

```diff
@@ -119,3 +127,26 @@ def plan():
         RE(plan())
 
     mock.assert_not_called()
+
+
+async def test_set_num_periods_works(RE, simpledae):
+    set_mock_value(simpledae.number_of_periods, 1)
+    RE(set_num_periods(simpledae, 5))
+    assert (await simpledae.number_of_periods.get_value()) == 5
+
+
+def test_set_num_periods_fails(RE, simpledae):
+    class UnequalInt(int):
```

Comment:
fixture for this? i could see it being helpful for other tests

Reply:
This whole thing has gone away since we no longer have a `set_num_periods` plan stub